rfcs/rfcOVID-0018.md

```diff
@@ -0,0 +1,104 @@
+# A feature to automatically "yield true" at the end of a file
+
+## Preamble
+
+    Author:  Curtis "Ovid" Poe <curtis.poe@gmail.com>
+    Sponsor:
+    ID:      OVID-0018
+    Status:  Draft
+
+## Abstract
+
+This RFC proposes a `yield_true` feature. When used, the current Perl _file_
+containing `use feature 'yield_true';` will automatically return a true value
+after successful compilation, eliminating the need for a "1" (or other true
+value) at the end of the file.
+
+## Motivation
+
+Eliminate the need for a true value at the end of a Perl file.
+
+## Rationale
+
+There's no need to have a true value be hard-coded in our files that we
+`use`. Further, newer programmers can get confused because sometimes code
+_doesn't_ end with a true value but nonetheless compiles just fine because
+_something_ in the code returned a true value and the code compiles as a
+side-effect.
```

Comment:
It is my understanding that `use`, `do`, and `require` actually execute code. Sure there is the "compile" phase accessible via blocks like `BEGIN` but modules are not _loaded_ in some `perl -c` kind of way as far as I undertand it. So I think this might affect the whole point of this - a return code of some kind is actually required _or_ whatever is loading the code should be sandboxed in some kind of `eval` in order to keep the calling program from `die`'ing if things go sideways.